rcgen/src/key_pair.rs

```diff
@@ -101,12 +101,25 @@ impl KeyPair {
 					serialized_der: key_pair_serialized,
 				})
 			},
+			#[cfg(all(feature = "aws_lc_rs", not(feature = "ring")))]
+			SignAlgo::Rsa(sign_alg) => {
+				use aws_lc_rs::encoding::AsDer;
+				let key_pair_doc =
```

Comment:
What's the meaning of the "_doc" suffix on this var? I think it could be in-lined into L109 in either case.

Reply:
"document". I tried keeping the code analogous to the other match cases.

Reply:
> I tried keeping the code analogous to the other match cases.

Sounds reasonable to me :+1: